<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget_simple</class>
 <widget class="QDockWidget" name="DockWidget_simple">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>326</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>346</width>
    <height>268</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Data gather plugin</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="startRecording_button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Start recording</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="outputFreeSpace_label">
      <property name="text">
       <string>Free disk space: ? MB</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="recording_groupBox">
      <property name="title">
       <string>Recording sources</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="GpsStatusWidget" name="gps_widget" native="true">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="gps_label">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>GPS fix available (using ? satellites out of ?)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="audio_checkBox">
         <property name="text">
          <string>Audio recording</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="video_checkBox">
         <property name="text">
          <string>Video recording</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="full_pushButton">
      <property name="text">
       <string>Switch to full interface</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GpsStatusWidget</class>
   <extends>QWidget</extends>
   <header>GpsStatusWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
